base/combinatorics.jl

```diff
@@ -59,6 +59,8 @@ function factorial(n::Integer)
     return f
 end
 
+factorial(x::Number) = gamma(x + 1) # fallback for x not Integer
```

Comment:
I thought `x` was a good visual indication that the argument here need not be an integer.